dynawo/sources/Modeler/ModelManager/test/Test.cpp

```diff
@@ -235,9 +241,12 @@ class MyModelica: public ModelModelica {
    * @param yp values of the derivatives of the continuous variable
    * @param F computes values of the residual functions
    */
-  virtual void evalFAdept(const std::vector<adept::adouble> &y, const std::vector<adept::adouble> &yp, std::vector<adept::adouble> &res) {
+  virtual void evalFAdept(const std::vector<adept::adouble> &y, const std::vector<adept::adouble> &yp,
+    const std::vector<adept::adouble> &yExt, const std::vector<adept::adouble> &ypExt, std::vector<adept::adouble> &res) {
     ASSERT_EQ(y.size(), 2);
     ASSERT_EQ(yp.size(), 2);
+    ASSERT_EQ(yExt.size(), 1);
+    ASSERT_EQ(ypExt.size(), 1);
     ASSERT_EQ(res.size(), 2);
     res[0] = 2*y[0]+yp[1];
```

Comment:
Maybe we could use the external variables in the evalF to make sure everything is OK

Reply:
OK